core/src/test/scala/org/broadinstitute/dsde/rawls/jobexec/MethodConfigResolverSpec.scala

```diff
@@ -18,121 +18,127 @@ class MethodConfigResolverSpec extends WordSpecLike with Matchers with TestDrive
+        |
+        |workflow w1 {
+        |  Array[Array[Int]] aint_array
+        |  scatter(ai in aint_array) {
+        |    call t1 { input: aint_arg = ai }
+        |  }
+        |}
+      """.stripMargin)
 
 
   val optionalDoubleArrayWdl =
-    """
-      |task t1 {
-      |  Array[Int] aint_arg
-      |  command {
-      |    echo ${aint_arg}
-      |  }
-      |}
-      |
-      |workflow w1 {
-      |  Array[Array[Int]]? aint_array
-      |  scatter(ai in aint_array) {
-      |    call t1 { input: aint_arg = ai }
-      |  }
-      |}
-    """.stripMargin
+    WdlSource(
+      """
+        |task t1 {
+        |  Array[Int] aint_arg
+        |  command {
+        |    echo ${aint_arg}
+        |  }
+        |}
+        |
+        |workflow w1 {
+        |  Array[Array[Int]]? aint_array
+        |  scatter(ai in aint_array) {
+        |    call t1 { input: aint_arg = ai }
+        |  }
+        |}
+      """.stripMargin)
 
 
   val tripleArrayWdl =
-    """
-      |task t1 {
-      |  Array[Array[Int]] aint_arg
-      |  command {
-      |    echo ${aint_arg}
-      |  }
-      |}
-      |
-      |workflow w1 {
-      |  Array[Array[Array[Int]]] aaint_array
-      |  scatter(ai in aaint_array) {
-      |    call t1 { input: aint_arg = ai }
-      |  }
-      |}
-    """.stripMargin
+    WdlSource(
+      """
+        |task t1 {
+        |  Array[Array[Int]] aint_arg
+        |  command {
+        |    echo ${aint_arg}
+        |  }
+        |}
+        |
+        |workflow w1 {
+        |  Array[Array[Array[Int]]] aaint_array
+        |  scatter(ai in aaint_array) {
+        |    call t1 { input: aint_arg = ai }
+        |  }
+        |}
+      """.stripMargin)
 
   val wdlVersionOneWdl =
-    """
-      |version 1.0
-      |
-      |task use_this_name {
-      |
-      |  input {
-      |    String s
-      |    File f
-      |  }
-      |
-      |  command {}
-      |
-      |  meta {
-      |    email: "skroob@spaceballs.gov"
-      |    author: "President Skroob"
-      |    description: "Spaceballs: The Unit Test"
-      |  }
-      |
-      |  output {
-      |    File f2 = "a"
-      |  }
-      |
-      |  runtime {
-      |    docker: "docker image"
-      |  }
-      |}
-    """.stripMargin
-
-  val badWdl = littleWdl.replace("workflow", "not-a-workflow")
+    WdlSource(
+      """
+        |version 1.0
+        |
+        |task use_this_name {
+        |
+        |  input {
+        |    String s
+        |    File f
+        |  }
+        |
+        |  command {}
+        |
+        |  meta {
+        |    email: "skroob@spaceballs.gov"
+        |    author: "President Skroob"
+        |    description: "Spaceballs: The Unit Test"
+        |  }
+        |
+        |  output {
+        |    File f2 = "a"
+        |  }
+        |
+        |  runtime {
+        |    docker: "docker image"
+        |  }
+        |}
+      """.stripMargin)
+
+  val badWdl = WdlSource("This is not a valid workflow [MethodConfigResolverSpec]")
```

Comment:
There are so many WDLs flying around in these test suites that I had to label where they came from to make any sense of it